coalib/bears/requirements/JuliaRequirement.py

```diff
@@ -29,3 +30,22 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'julia', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+       Creates the installation command for the instance of the class.
+
+       >>> JuliaRequirement("Lint", '', '-e').install_command()
+       'julia -e "Pkg.add(\"Lint\")'
+
+       :param return: A string with the installation command.
+       """
+        return 'julia {} "Pkg.add(\"{}\")'.format(self.flag, self.package)
+
+    def is_installed(self):
+        """
+       Checks if the dependency is installed.
+
+       :param return: True if dependency is installed, false otherwise.
+       """
+        return not call_without_output(('julia', self.flag, '"bad_package"'))
```

Comment:
hey this is bad. you are not even using the command you found, how are you going to test it exists or it doesnt?
also, why are you using ``bad_package`` here.. ? it should work for any package, as in have ``self.package`` instead.

what this command (call_without_output) does, is form a shell command given by the arguments.
so ``call_without_output(('rm', '-R', '-f', 'directoryA'))`` will run ``rm -R -f directoryA`` in the shell.
you basically want it to run the command you found, ``julia -e "pkg.installed(pkg_name)"`` and to work for any package.